storage/innobase/handler/ha_innodb.cc

```diff
@@ -11196,7 +11196,11 @@ and set the encryption flag in table flags
 dberr_t create_table_info_t::enable_master_key_encryption(dict_table_t *table) {
   const char *encrypt = m_create_info->encrypt_type.str;
 
-  if (Encryption::is_none(encrypt)) return (DB_SUCCESS);
+  /* if table is part of tablespace - no need for retrieving
+   * master key as tablespace key was already decrypted
```

Comment:
Please don't use this leading * column comment style

Reply:
sorry, keep forgetting